banzai/tests/test_header_checker.py

```diff
@@ -0,0 +1,127 @@
+from banzai.tests.utils import FakeImage
+from banzai.qc import header_checker
+
+
+import mock
+import numpy as np
+
+def test_no_input_images():
+    tester = header_checker.HeaderSanity(None)
+    images = tester.do_stage([])
+    assert len(images) == 0
+
+
+def test_group_by_keywords():
+    tester = header_checker.HeaderSanity(None)
+    assert tester.group_by_keywords is None
+
+def test_no_exptime_in_frames():
+    tester = header_checker.HeaderSanity(None)
+    tester.logger.error = mock.MagicMock()
+    nx = 101
+    ny = 103
+
+    images = [FakeImage(nx=nx, ny=ny) for x in range(6)]
+
+    for image in images:
+
+        tester.check_header_keyword_present('EXPTIME', image)
+        assert tester.logger.error.called
+
+    assert len(images) == 6
```

Comment:
We decided to not reject frame form this module. Just to indicates logs. We can change this if needed now, but this needs discussions.